crates/matrix-sdk-ui/src/timeline/event_handler.rs

```diff
@@ -181,14 +179,19 @@ pub(super) enum TimelineEventKind {
 
 impl TimelineEventKind {
     /// Creates a new `TimelineEventKind` with the given event and room version.
```

Comment:
We no longer provide a room version, and this is the place to document the arguments.

```suggestion
    /// Creates a new `TimelineEventKind`.
    ///
    /// # Arguments
    ///
    /// * `event` - The event for which we should create a `TimelineEventKind`.
    /// * `raw_event` - The [`Raw`] JSON for `event`. (Required so that we can access `unsigned` data.)
    /// * `room_data_provider` - An object which will provide information about the room containing the event.
    /// * `unable_to_decrypt_info` - If `event` represents a failure to decrypt, information about that failure. Otherwise, `None`.
```